lib/classes/ConfigSchemaHandler/errorMessageBuilder.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+const _ = require('lodash');
+
+const ANY_OF_EVENT_SCHEMA_PATH =
+  '#/properties/functions/patternProperties/%5E%5Ba-zA-Z0-9-_%5D%2B%24/properties/events/items/anyOf';
+
+// matches '#/properties/functions/patternProperties/%5E%5Ba-zA-Z0-9-_%5D%2B%24/properties/events/items/anyOf/123/additionalProperties'
+const ANY_OF_SUB_ERROR_REGEX = /^#\/properties\/functions\/patternProperties\/%5E%5Ba-zA-Z0-9-_%5D%2B%24\/properties\/events\/items\/anyOf\/[\d]+\/additionalProperties$/;
+
+// matches '#/properties/functions/patternProperties/%5E%5Ba-zA-Z0-9-_%5D%2B%24/properties/events/items/anyOf/17/properties/yourPluginEvent/properties/anotherProp/type'
+const DEEPER_ANY_OF_SUB_ERROR_REGEX = /^#\/properties\/functions\/patternProperties\/%5E%5Ba-zA-Z0-9-_%5D%2B%24\/properties\/events\/items\/anyOf\/[\d]+\/properties\/.+$/;
+
+/*
+ * Structure AJV errors to prevent secondary errors related to 'anyOf' condition
+ * are not shown. Those errors are placed in `suberrors` array of the most
+ * relevant error.
+ */
+const structureAjvErrors = ajvErrors => {
+  // If theres is no 'anyOf' error at schemaPath that equals to
+  // then do nothing and return ajvErrors
+  if (!ajvErrors.filter(err => err.schemaPath === ANY_OF_EVENT_SCHEMA_PATH).length) {
```

Comment:
More optimal would be to use `some` (we can also `for..of` and return in body)

Reply:
Thanks, I didn't know that!

Reply:
As for `for..of`, not sure I understand what you mean here

Reply:
it was more a stike suggestion, that instead of using `some`, it could also be constructed with `for..of` as:

```javascript
for (const err of ajvErrors) {
  if (err.schemaPath === ANY_OF_EVENT_SCHEMA_PATH) return;
}
```

Reply:
Yeah, this seem to work too. I hope this is not a blocker and the adjustments made are enough to start testing

Reply:
Yes, don't worry about that, it's obviously not a blocker.

I will pursue testing if not today, then on Friday

Reply:
Thanks for letting me know